.github/workflows/pre-commit.yml

```diff
@@ -1,26 +0,0 @@
-name: pre-commit
```

Comment:
So... what's this removal doing? Doesn't seem to be related to code-cov? Worth a comment in the PR.

Reply:
Yeah my apologies, added a description to the PR description. But essentially we moved this to the new azure workflow file, and it's still (correctly) being picked up as a required task since the name is the same